src/cmake/build_client.cmake

```diff
@@ -26,8 +26,11 @@ if(BUILD_CLIENT)
             message(STATUS "osxcross build, building main.m for client")
             list(APPEND mac_client_sources xcode/main.m)
         endif()
-
         list(APPEND client_sources ${mac_client_sources})
+    elseif(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
```

Comment:
Can you please remove the first `if`? We have it twice in here now, but once should be enough with this `elseif` clause.